jazzy/distribution.yaml

```diff
@@ -6196,6 +6196,21 @@ repositories:
       url: https://github.com/ros/robot_state_publisher.git
       version: jazzy
     status: maintained
+  robot_upstart:
+    doc:
+      type: git
+      url: https://github.com/clearpathrobotics/robot_upstart.git
+      version: foxy-devel
+    release:
+      tags:
+        release: release/jazzy/{package}/{version}
+      url: https://github.com/clearpath-gbp/robot_upstart-release.git
+      version: 1.0.3-1
+    source:
+      type: git
+      url: https://github.com/clearpathrobotics/robot_upstart.git
+      version: foxy-devel
```

Comment:
I'm assuming that `foxy-devel` here for Jazzy is intentional, since it's also used for Galactic and Humble.

Reply:
That's correct; there were no changes needed since the Foxy release, so the same branch is still being used for all ROS 2 releases.